### DISCUSSION TITLE
"overflow" buffer type not ready for production?

### DISCUSSION THREAD

**Discussion by leshow** [vectordotdev/vector#21146] [Q&A]
The docs mention "Overflow buffers are not yet suitable for production workloads and may contain bugs that ultimately lead to data loss."

https://vector.dev/docs/about/under-the-hood/architecture/buffering-model/

Is this statement still accurate? What could be done to make overflow buffer configurations production-ready?

**Comment by jszwedko** [comment]
Hey! Yes, that statement is still accurate. We'd like to see more beta usage of the feature for real-world testing before calling it "production-ready".
